conformance/google-javalite/build.gradle

```diff
@@ -25,7 +25,7 @@ sourceSets {
 dependencies {
     implementation(libs.kotlin.coroutines.core)
     implementation(libs.protobuf.kotlin)
-    implementation(project(":crosstests:common"))
+    implementation(project(":conformance:common"))
```

Comment:
This file is losing the `.kts` extension as part of the rename (which looks like it might be why the CI workflows are failing).